l10n.py

```diff
@@ -234,26 +220,13 @@ def file(self, lang: str, plugin_path: str | None = None) -> TextIO | None:
             except OSError:
                 logger.exception(f'could not open {file_path}')
 
-        elif getattr(sys, 'frozen', False) and sys.platform == 'darwin':
-            res_path = join(self.respath(), f'{lang}.lproj', 'Localizable.strings')
-            return open(res_path, encoding='utf-16')
-
         res_path = join(self.respath(), f'{lang}.strings')
         return open(res_path, encoding='utf-8')
 
 
 class _Locale:
     """Locale holds a few utility methods to convert data to and from localized versions."""
 
-    def __init__(self) -> None:
```

Comment:
Removal of __init__() intentional?

Reply:
Yes. The init was required because some settings had to be set on class creation if/only if Darwin. This is not the case so not needed.